index.html

```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
+            <ul class="text">
+              <li class="text-ruby">Ruby on Rails</li>
+              <li class="text-css">CSS</li>
+              <li class="text-java">JavaScript</li>
+              <li class="text-html">HTML</li>
+            </ul>
+            <div class="button1"><button class="button-text" type="button">See project </button></div>
+          </div>
+      </div>
+      <div  class="placeholder-container">
+          <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+          <div class="text-container">
+            <h1>Multi-Post Stories <br> Gain+Glory</h1>
+            <ul class="text">
+              <li class="text-ruby">Ruby on Rails</li>
+              <li class="text-css">CSS</li>
+              <li class="text-java">JavaScript</li>
+              <li class="text-html">HTML</li>
+            </ul>
+            <div class="button1"><button class="button-text" type="button">See project </button></div>
+          </div>
+      </div>
+      <div  class="placeholder-container">
+        <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+        <div class="text-container">
+          <h1>Multi-Post Stories <br> Gain+Glory</h1>
+          <ul class="text">
+            <li class="text-ruby">Ruby on Rails</li>
+            <li class="text-css">CSS</li>
+            <li class="text-java">JavaScript</li>
+            <li class="text-html">HTML</li>
+          </ul>
+          <div class="button1"><button class="button-text" type="button">See project </button></div>
+        </div>
+    </div>
+    </section>
+      <section class="about-container">
+        <div class="text-aline">
+  
+            
+        
+          </div>
+          <div class="text-about">
+            <div class="plane-container">
+              
+              <h3>About me</h3>
+              <div class="plane-image">
+                <img src="images/plane.svg" alt="plane">
+              </div>
+            </div>
+           
+            <div class="test2">
+              <div>
+                <p class="h6"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                  don’t hesitate to contact me.</p>
+              </div>
+              
+            </div>
+              <div class="button2">
+                <button class="button-resume" type="button">Get my resume </button>
+              </div>
+          </div>
+          <div class="plane">
+            <img src="images/square.svg" alt="square">
+          </div>
+        </div>
+  
+         <div class="section6">
+          <article class="qoute">
+            <div class="holder-text">
+              <img src="images/holder.svg" alt="holder">
+              <div class="holder-text1">
+                <img src="images/ellipse.svg" alt="holder">
+                <div class="Language">
+                  <p>Language</p>
```

Comment:
- [ ] Kindly make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li